recipes/nanodbc/all/conanfile.py

```diff
@@ -12,7 +14,6 @@ class NanodbcConan(ConanFile):
     homepage = "https://github.com/nanodbc/nanodbc/"
     url = "https://github.com/conan-io/conan-center-index"
     settings = "os", "arch", "compiler", "build_type"
```

Comment:
I think that we could add the `package_type` here:

```python
package_type = "library"
```

Reply:
Very right! Just added this